### PR TITLE
closes #27

### LINKED ISSUE #27: Model Tests
Writing Model Tests!

### PR DESCRIPTION
**Pull request by notmarkmiranda**
# Closes #27 
## Primary
- Add model tests for all models
## Secondary
- Add SimpleCov
## Misc
- All tests pass: 64
- Test coverage: 82.26%

### CODE REVIEW

File: spec/models/business_spec.rb [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+    it { should have_many :business_categories }
+    it { should have_many(:categories).through(:business_categories) }
+    it { should have_many(:event_spaces) }
+    it { should have_many(:applications) }
+    it { should have_many(:accepted_events).through(:event_spaces) }
+    it { should have_many(:applied_events).through(:applications) }
 
+    # shoulda matchers does support a uniqueness validation with the association yet
+    # https://github.com/thoughtbot/shoulda-matchers/issues/814
+    # it { should validate_uniqueness_of(:business_name).scoped_to(:vendor) }
```
Comment (notmarkmiranda) [inline spec/models/business_spec.rb:15]: this is the only weird thing I ran into @JaredRoth. I can remove this comment, but didn't want it to be a weird thing.

Comment (JaredRoth) [reply spec/models/business_spec.rb:15]: At this point, I would just leave it.

**Comment by JaredRoth** [comment]
Thanks for getting all this started. I had held off from model level testing up to this point because things have been drastically changing and I knew they would have a lot more methods on them eventually. But this helps by giving me a foundation to build on rather than _another_ thing I have to build from scratch. Thanks.

File: spec/models/faire_spec.rb [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+    it { should validate_presence_of :city }
+    it { should validate_presence_of :admin }
+  end
+
+  context "methods" do
+    it "#full_title - adds the name of the city to the title" do
+      user = create(:admin)
+      faire = create(:faire, admin: user)
+
+      expect(faire.full_title).to include(faire.city.name)
```
Comment (JaredRoth) [inline spec/models/faire_spec.rb:19]: Changed this to test how #full_title is different from #title. (And made the test title explicit)

**Comment by JaredRoth** [comment]
Unless you (@notmarkmiranda) see a problem with few changes I made, I'll merge it.

**Pull request merged**